projects/dev-schematics/.npmignore

```diff
@@ -0,0 +1,5 @@
+# Ignores TypeScript files, but keeps definitions.
+*.ts
```

Comment:
Let's ignore spec files as well

Reply:
Same applies to schematics lib. Why publish the spec files to npm 🤦

Reply:
Good point, this should be ignored in both `schematics` and `dev-schematics`. However, we don't plan to publish the `dev-schematics` to npm

Reply:
Done

Reply:
For spec files we also have `*_spec.js` and `*_spec.js.map`